specs/fulu/p2p-interface.md

```diff
@@ -209,6 +209,8 @@ The following validations MUST pass before forwarding the `sidecar: DataColumnSi
 
 Honest nodes SHOULD query `engine_getBlobsV2` as soon as they receive a valid `beacon_block` or `data_column_sidecar` from gossip. If ALL blobs matching `kzg_commitments` are retrieved, they should convert the response to data columns, and import the result.
 
+Clients SHOULD assume the execution layer is trusted and skip verification for blobs retrieved from it.
```

Comment:
Personally, I think it could be good to incorporate this into this part (two lines below)
>  When clients use the local execution layer to retrieve blob and compute data columns, they MUST behave as if the imported `data_column_sidecar` had been received via gossip. In particular, clients MUST:
→ it MUST behave ~ received via gossip, except ~

Reply:
I might have misunderstood what you meant, but I revised it to combine with the section below. Let me know if you think that looks good, otherwise I'm happy to iterate on it more.

Reply:
I like the new version. And I agree this is the appropriate spot for the note.